### PR TITLE
US117908: beta version of new engagement dashboard

### PR DESCRIPTION
**Pull request by johngwilkinson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     "d2l-consistent-evaluation": "BrightspaceHypermediaComponents/consistent-evaluation#semver:^0",
     "d2l-content-store": "Brightspace/d2l-content-store#semver:^0",
     "d2l-cpd": "Brightspace/continuous-professional-development#semver:^1",
-    "d2l-engagement-dashboard": "Brightspace/insights-engagement-dashboard#semver:^0",
+    "d2l-engagement-dashboard": "Brightspace/insights-engagement-dashboard#487936c",
```
Comment (johngwilkinson) [inline package.json:61]: For some reason I had to specify the commit to get npm to pull the latest release (v0.1.1)

Comment (Vitalii-Misechko) [reply package.json:61]: `Brightspace/insights-engagement-dashboard#semver:^0.1` - did the trick on my local env.

image

IIRC, semver before v1.0 behaves with a set of a bit different rules.

**Review by Vitalii-Misechko — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (johngwilkinson) [reply package.json:61]: That works, thanks. CI discovered that other packages are using a different version of lit-mobx, so I'll need to change that in our repo then make another PR here.

**Comment by johngwilkinson** [comment]
Fixed package conflict

**Pull request merged**